mne_bids/dig.py

```diff
@@ -269,11 +271,20 @@ def _coordsystem_json(*, raw, unit, hpi_coord_system, sensor_coord_system,
                          .format(coord_frame))
 
     # get the coordinate frame description
+    try:
+        sensor_coord_system, sensor_coord_system_mne = sensor_coord_system
+    except ValueError:  # pragma: no cover
```

Comment:
why `no cover`?

Reply:
initially I thought this is never reached in our code base, but I was wrong. I'll remove this pragma